src/ui/public/doc_table/components/table_row.js

```diff
@@ -77,6 +78,35 @@ module.directive('kbnTableRow', function ($compile) {
 
         $detailsScope.row = $scope.row;
 
+        $detailsScope.copyTextToClipboard = text => {
```

Comment:
Ok I'll give it a shot. Where should it go and what should it be named? I have a lib I created called [copee](https://www.npmjs.com/package/copee). Maybe I can make that compatible with angular's import and include it as a dependency?